plugins/com.google.cloud.tools.eclipse.welcome/intro/cloud-tools-for-eclipse.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<introContent>
+  <extensionContent path="overview/@" style="css/intro.css">
+    <group style-id="content-group">
```

Comment:
When installed, the Welcome will be opened on the _What's New_ page (`path="whatsnew/@"`), and the new contributions are subtly highlighted on the page.

There's sometimes a Migration page offered too (`path="migrate/@"`) and we could link to the _Migrating from GPE_ page.